<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>695</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="checksPushButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>240</y>
      <width>100</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Checks</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>170</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="apiKeyLabel">
       <property name="text">
        <string>API Key</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="apiKeyLineEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QComboBox" name="tokenComboBox">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>250</y>
      <width>85</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="downtimesPushButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>290</y>
      <width>100</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>downtimes</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>269</width>
      <height>62</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="maximumSize">
        <size>
         <width>22</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="qupdownio-ui-res.qrc">:/images/updown-logo.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="maximumSize">
        <size>
         <width>35</width>
         <height>60</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font-family: &quot;Oleo Script&quot;,cursive;
font-style: italic;
font-weight: normal;
font-size: 24pt;
color: #6b0;
text-shadow: 0 1px 0 #fff,0 -1px 0 #457e00;
</string>
       </property>
       <property name="text">
        <string>Q</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>60</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #555;
font-style: italic;
font-family: &quot;Oleo Script&quot;,cursive;
font-size: 24pt;
font-weight: normal;
text-shadow: 0 1px 0 #fff,0 -1px 0 #000;
</string>
       </property>
       <property name="text">
        <string>Updown</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>60</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font-family: &quot;Oleo Script&quot;,cursive;
font-style: italic;
font-weight: normal;
font-size: 24pt;
color: #6b0;
text-shadow: 0 1px 0 #fff,0 -1px 0 #457e00;
</string>
       </property>
       <property name="text">
        <string>.io</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="menuWidget" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid white;</string>
    </property>
    <widget class="QLabel" name="arrowPixmapLabel">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>10</y>
       <width>20</width>
       <height>15</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="qupdownio-ui-res.qrc">:/images/arrow-right.png</pixmap>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>230</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="apiKeyLabel_2">
       <property name="text">
        <string>Host</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="hostLineEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="addCheckPushButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>360</y>
      <width>100</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Add check</string>
    </property>
   </widget>
   <widget class="QPushButton" name="updateCheckPushButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>370</y>
      <width>100</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Update check</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteCheckPushButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>390</y>
      <width>100</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete check</string>
    </property>
   </widget>
   <widget class="QPushButton" name="settingsPushButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>430</y>
      <width>100</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Settings</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>695</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qupdownio-ui-res.qrc"/>
 </resources>
 <connections/>
</ui>
